packages/apollo-server/README.md

```diff
@@ -63,25 +69,21 @@ server.listen().then(({ url }) => {
 });
 ```
 
-> Due to its human-readability, we recommend using [schema-definition language (SDL)](https://www.apollographql.com/docs/apollo-server/essentials/schema.html#sdl) to define a GraphQL schema but [a `GraphQLSchema` object from `graphql-js`](https://github.com/graphql/graphql-js/#using-graphqljs) can also be specified in place of `typeDefs` and `resolvers` using the `schema` property:
+> Due to its human-readability, we recommend using [schema-definition language (SDL)](https://www.apollographql.com/docs/apollo-server/essentials/schema.html#sdl) to define a GraphQL schema--[a `GraphQLSchema` object from `graphql-js`](https://github.com/graphql/graphql-js/#using-graphqljs) can also be specified instead of `typeDefs` and `resolvers` using the `schema` property:
 >
 > ```js
 > const server = new ApolloServer({
 >   schema: ...
 > });
 > ```
 
-Finally, start the server using `node index.js` and open your web-browser to the URL which is output on the console.
+Finally, start the server using `node index.js` and go to the URL returned on the console.
 
-For more details, check out the Apollo Server [Getting Started guide](https://www.apollographql.com/docs/apollo-server/getting-started.html) of the documentation, or for a more comprehensive understanding, see the [fullstack tutorial](https://www.apollographql.com/docs/tutorial/introduction.html).
-
-For questions, the [Apollo commuinty on Spectrum.chat](https://spectrum.chat) is a great place to get assistance.
+For more details, check out the Apollo Server [Getting Started guide](https://www.apollographql.com/docs/apollo-server/getting-started.html) and the [fullstack tutorial](https://www.apollographql.com/docs/tutorial/introduction.html).
 
 ## Installation: Integrations
 
-While the standalone installation above can be used without making a decision about which web framework to use, the Apollo Server integration packages are paired with specific web frameworks (e.g. Express, Koa, hapi).
-
-The following web frameworks have Apollo Server integrations, and each of these linked integrations has its own installation instructions and examples on its package `README.md`:
+While the standalone setup is quite straightforward--as it does not require a web framework--the integration packages have their own instructions (with respective examples). Please refer to the following guides to combine Apollo Server with a specific web framework.
```

Comment:
Let's avoid calling it `quite straightforward`.  The previous instruction did a good job explaining the benefits without making claims to simplicity.

In fact, I'd ask that we revert this back to how it was written before (albeit some smaller tweaks are still welcome, like removing the `README` reference, if you're so compelled.